widget/ui/src/containers/SwapInput/SwapInput.tsx

```diff
@@ -0,0 +1,69 @@
+import type { SwapInputProps } from './SwapInput.types';
+
+import React from 'react';
+
+import { Button, Typography } from '../../components';
+
+import { Container, InputAmount } from './SwapInput.styles';
+import { TokenSection } from './TokenSection';
+
+export function SwapInput(props: SwapInputProps) {
+  const { chainImage, tokenImage, chain, label, tokenSymbol, error } = props;
+  return (
+    <Container>
+      <div className="label__container">
+        <div className="label">
+          <Typography variant="body" size="small" color="$neutral400">
+            {label}
+          </Typography>
+          {label === 'From' && (
+            <div className="balance">
+              <Typography
+                color="$neutral400"
+                mr={4}
+                variant="body"
+                size="xsmall">
+                {props.balance}
+              </Typography>
+              <Button
+                variant="contained"
+                size="xsmall"
+                type="primary"
+                onClick={props.selectMaxBalance}>
+                Max
+              </Button>
+            </div>
+          )}
+        </div>
+      </div>
+      <div className="form">
+        <TokenSection
+          chain={chain}
+          tokenSymbol={tokenSymbol}
+          error={error}
+          chainImage={chainImage}
+          tokenImage={tokenImage}
+        />
+        <div className="amount">
+          <InputAmount
+            disabled={label === 'To'}
+            style={{ padding: 0 }}
+            value={label === 'From' ? props.inputAmount : props.outputAmount}
+            type="number"
+            size="large"
+            placeholder="0"
+            variant="ghost"
+            min={0}
+            {...(label === 'From' && {
```

Comment:
When an input `disabled`, it shouldn't trigger anything. you can remove this condition.